pkg/cli/iam_handle_test.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/iam/apiv1/iampb"
+	"github.com/abcxyz/access-on-demand/pkg/handler"
+	"github.com/abcxyz/access-on-demand/pkg/testutil"
+	"github.com/abcxyz/pkg/logging"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/genproto/googleapis/type/expr"
+	"google.golang.org/protobuf/testing/protocmp"
+
+	pkgtestutil "github.com/abcxyz/pkg/testutil"
+)
+
+func TestIAMHandleCommand(t *testing.T) {
+	t.Parallel()
+
+	// Set up IAM request file.
+	requestFileContentByName := map[string]string{
+		"valid.yaml": `
+policies:
```

Comment:
I believe the order in yaml are respected when unmarshalled to IAMRequest (struct of list of struct that contains list), also confirmed it with Bard, but somehow the CI failed because of order ( it is reran so the failed history was gone).